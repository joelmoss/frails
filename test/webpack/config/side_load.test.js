const createConfig = require('../../../package/webpack/create_config')

describe('config.sideLoad', () => {
  test.skip('boolean', () => {
    expect(createConfig({ sideload: false })).toMatchSnapshot()
  })
})
