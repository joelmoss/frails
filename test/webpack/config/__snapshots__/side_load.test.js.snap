// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config.sideLoad boolean 1`] = `
Object {
  "entry": "./app/assets/application.js",
  "mode": "development",
  "output": Object {
    "path": "/Users/joelmoss/dev/frails/test/dummy/public/assets",
    "publicPath": "/assets/",
  },
}
`;
